web/styles/popovers.css

```diff
@@ -1190,4 +1218,120 @@ ul {
         width: 16px;
         height: 16px;
     }
+
+    .zulip-icon {
+        position: relative;
+        top: -1px;
+    }
+}
+
+#gear-menu-dropdown {
+    padding: 5px 0;
+    box-shadow:
+        0 2.7798px 4.1129px 0 hsl(0deg 0% 0% / 10%),
+        0 5.5113px 8.5783px 0 hsl(0deg 0% 0% / 9%),
+        0 8.4377px 13.9271px 0 hsl(0deg 0% 0% / 11%),
+        0 12.177px 21.4737px 0 hsl(0deg 0% 0% / 11%),
+        0 18.7257px 35.4802px 0 hsl(0deg 0% 0% / 15%),
+        0 41px 80px 0 hsl(0deg 0% 0% / 20%);
+
+    .org-info {
+        padding-top: 4px;
+
+        & li {
+            display: flex;
+            justify-content: center;
+            font-size: 15px;
+            font-style: normal;
+            font-weight: 400;
+            line-height: 16px; /* 114.286% */
+            letter-spacing: 0.28px;
+
+            &:focus-within {
+                background: var(--color-background-hover-dropdown-menu);
+            }
+
+            & a {
```

Comment:
We should get out of the habit of having inner-most selectors as element selectors. The reason being is that browsers evaluate selectors from right to left, meaning that every time a selector ends in an element, the browser has to work that much harder whenever and wherever on a page it encounters the element.

I note in the template below that some of the anchors have a class of one sort or another, but it might be good--as with the topmost `<li>` elements, but for a different reason here--to have a shared class that you can reference here. That might also help get rid of the `!important` in the `:focus` pseudo-class below.

Reply:
We should probably start a #frontend discussion on this -- might be worth purging and then adding a `stylelint` rule to ban.

(I think this might be the kind of thing we'd find if we figured out how to profile our CSS)

Reply:
Doesn't help to remove `!important` here. Would you like me to use a hack like `.navbar-dropdown-menu.navbar-dropdown-menu` to get it removed. I am not sure which is better.

Reply:
https://chat.zulip.org/#narrow/stream/6-frontend/topic/CSS.20optimization for discussion